clabverter/clabverter.go

```diff
@@ -304,6 +304,12 @@ func (c *Clabverter) load() error {
 		return err
 	}
 
+	// set the destination namespace to the c9s-<topology name>
+	// if is was not explicitly set via the cli
+	if c.destinationNamespace == "" {
+		c.destinationNamespace = fmt.Sprintf("c9s-%s", c.clabConfig.Name)
```

Comment:
we probably want to use the `kubernetesutil.SafeConcatNameKubernetes` here in case the lab name is long

Reply:
I can fix this when I fix my test mess too